extensions/codeql-csharp-extensions/models/AspNetCoreMvc.yml

```diff
@@ -0,0 +1,10 @@
+extensions:
+  - addsTo:
+      pack: codeql/csharp-queries
+      extensible: sinkModel
+    data:
+      # XSS Requires the ContentType set to 'text/html'
+      - ["Microsoft.AspNetCore.Mvc", "ContentResult", False, "Content", "", "", "", "xss", "manual"]
+      - ["Microsoft.AspNetCore.Mvc", "ContentResult", False, "ContentType", "", "", "", "xss", "manual"]
+      # Returned with a ContentType of 'plain/text'
```

Comment:
will including this lead to FP for xss
# Returned with a ContentType of ‘plain/text’

Reply:
potentially but keeping in the model for now - idea is to run as an audit and find all the things